docs/src/tutorials/software-management.md

```diff
@@ -70,51 +41,30 @@ More info about managing the device software can be found [here](https://cumuloc
+   > Note: The version field format is `package_version::plugin_type_name`.The plugin type name that is provided here is used to pick
+     the appropriate plugin among those installed in /etc/tedge/sm-plugins.
 3. Optionally, you can define the device type filter when adding a new software.
-4. As of now thin-edge.io only supports software from apt repositories. So, select the **Provide a file path file path** option and give an **empty space**.
+4. As of now thin-edge.io only supports software from apt repositories. So, select the **Provide a file path** option and give an **empty space**.
 5. Click Save.
 
 Follow above mentioned steps as shown in the diagram
 
  ![Add new software](./images/add-new-software-to-repo.png)
 
 ### Deleting software or software version
-
-Find more information about how to delete the software or the specific software version [here](https://cumulocity.com/guides/users-guide/device-management/#deleting-softwares-or-software-versions)
+One can remove a software module or a specific version of it from the software repository.
+Find more information about [how to delete the software or the specific software version](https://cumulocity.com/guides/users-guide/device-management/#deleting-softwares-or-software-versions)
 
 
 
 ## Managing software on a device
-
 Find more information about [how to manage the software](https://cumulocity.com/guides/users-guide/device-management/#managing-software-on-a-device) on a device.
 
-### Install software on a device
-Find about how to [Install a specific software on a device](https://cumulocity.com/guides/users-guide/device-management/#to-install-software-on-a-device)
-> Note: Software profiles are not supported as of now on thin-edge.
-
-### Update software on a device
-Find about how to [Update the software on a device](https://cumulocity.com/guides/users-guide/device-management/#to-update-software-on-a-device)
-
-### Remove software on a device
-Hover over the software entry which you want to remove and click the **Remove** software icon and click on `Apply changes` button.
-
-
-
-## Starting and Stopping the Software management services manually
-
-For debugging purpose the services can be started/stopped manually as shown below.
-
-### Starting the services manually
-
-```shell
-sudo systemctl start tedge-agent.service
-sudo systemctl start tedge-mapper-sm-c8y.service
-```
-
-### Stopping the services manually
-
-```shell
-sudo systemctl stop tedge-agent.service
-sudo systemctl stop tedge-mapper-sm-c8y.service
-```
+From the Cumulocity cloud `Software` tab of a device, software can be
+* [installed](https://cumulocity.com/guides/users-guide/device-management/#to-install-software-on-a-device)
+ > Note: Software profiles are not supported as of now on thin-edge.
+* [updated](https://cumulocity.com/guides/users-guide/device-management/#to-update-software-on-a-device)
+ > Note: Thin-edge treats install and update same.
+* removed
```

Comment:
Provide this hyperlink: https://cumulocity.com/guides/users-guide/device-management/#to-delete-software-from-a-device for remove as well just like the other options?